frontend/src/components/widgets/CardBadges.js

```diff
@@ -38,7 +42,17 @@ export default ({ card }) => {
     codeReviewNyCompetentSinceLastReviewRequest,
     codeReviewRedFlagSinceLastReviewRequest,
     openPrCount,
+    oldestOpenPrUpdatedTime,
   } = card;
+
+  let ageOfOpenPR = () => {
```

Comment:
This function can be a bit cleaner in a few ways. Firstly, use consts for constants

Now looking at this line:
```
    const age = timeAgo.format(date - 24 * 60 * 60 * 1000);
```
Why are you subtracting that number? This line deserves a comment. And this function deserves some tests.

When adding comments you can describe why you did something (so that the next programmer doesn't waste time second-guessing your work). You can also link to external documentation. Eg you can say `// based on https://something.useful`.

Unit tests serve multiple purposes: One is to test the code (dah) but also good tests act as documentation. Good tests describe the befavious of the code.  If you write tests for this function you would need to refactor your code so that this function is testable. Perhaps put it inside a utils.js script and let it accept the date as a parameter. eg:

```
getAgeString = ({dateString}) => { // notice the name changes. Why do you think I would suggest this?
      const date = new Date(dateString);
      const timeAgo = new TimeAgo('en-US');
      // useful comment goes here
      return timeAgo.format(date - 24 * 60 * 60 * 1000); 
}
```

Remember that any fool can make code that runs. A good coder writes code that people can read.